src/dom.js

```diff
@@ -367,15 +367,20 @@ export function childElementByTag(parent, tagName) {
  * @param {!Element} element
  * @param {function(string):string} opt_computeParamNameFunc to compute the parameter
  *    name, get passed the camel-case parameter name.
+ * @param {string} opt_attrPrefix if it's something different than the default 'data-param',
+ *    i.e. 'data-' or 'data-foo-'
  * @return {!Object<string, string>}
  */
-export function getDataParamsFromAttributes(element, opt_computeParamNameFunc) {
+export function getDataParamsFromAttributes(element,
```

Comment:
@cramforce is that cool? The use case here is that all of our widgets support `data-<something>` and not `data-params-<something>`, I had a [function](https://github.com/cXense/amphtml/blob/5e754a9141955005e5bd4fa031f403300035dc2c/extensions/amp-cxense-player/0.1/amp-cxense-player.js#L152-L163) that does that for me, but I'd rather use your utilities, and extend them if needed.

IMO, I don't think the function should just keep adding optional arguments in the signatures, I'd rather just have an `options` hash for all optional arguments, but that doesn't seem like the convention followed in a lot of the files in `src/`